docs/conda_windows_install.md

```diff
@@ -17,3 +17,71 @@ conda deactivate
 ```
 
 To use your environment, run the command "conda activate popsborder_env" or select the popsborder_env environment from your IDE (e.g. VS Code, Atom, PyCharm).
+
+## Adding Conda Environments to Jupyter Notebook
+
+ Conda allows you to create isolated environments with specific 
+ versions of Python and its packages. This prevents conflicts between 
+ different projects' dependencies.
+
+### Step 1: Create your conda environment.
+Inside of your conda based command line navigate inside your forked PopsBorder
+folder. Next use the following command
+```bat
+conda create -n PopsBorderEnv python=3.8
+```
+This creates your environment with python version 3.8.
+PopsBorderEnv is the name of your environment. Change as needed.
+
+### Step 2: Activate your environment.
+Using your newly created environment from the previous step, run
+```bat
+conda activate PopsBorderEnv
+```
+Now you have entered into your environment.
+
+### Step 3: Install pipenv into you environment.
+Now install pipenv into your environment, a packaging tool we utilize
+for our project.
+```bat
+pip install pipenv
```

Comment:
Why pip install now? Before it was conda install. Same for ipykernel below.

Reply:
All tutorials I looked up used pip. Would you advise conda?

Reply:
All tutorials I found used pip. Would you advise conda instead?

Reply:
You are in a conda environment here, so it would make sense to use that. See if the tutorials used a conda environment.

Reply:
They were,  but I can try with conda instead.